src/API/src/db/occurrence/occurrence.resolver.ts

```diff
@@ -34,6 +37,9 @@ export const booleanTypeResolver = () => Boolean;
 @ObjectType()
 class PaginatedOccurrenceData extends PaginatedResponse(Occurrence) {}
 
+@ObjectType()
+class PaginatedOccurrenceCsvData extends PaginatedResponse(String) {}
```

Comment:
Can you rename this to `PaginatedStringData `? That way it can be reused for other paginated string data, if we have any.